components/dashboard/Form.js

```diff
@@ -2,15 +2,16 @@ import { useEffect, useMemo, useState } from 'react'
 import { territoryNames } from 'country-util'
 import { useForm, Controller } from 'react-hook-form'
 import { Box, Flex } from 'ooni-components'
-import moment from 'moment'
 import { MultiSelect } from 'react-multi-select-component'
 import { useIntl } from 'react-intl'
+import moment from 'moment'
+import dayjs from 'services/dayjs'
```

Comment:
Is it a feature of next.js to allow absolute imports relative to the components directory tree?

Reply:
It requires a bit of configuration, which is already there https://github.com/ooni/explorer/blob/master/jsconfig.json#L2-L4